### PR TITLE
feat: UI Kit 2.0 Byline

### PR DESCRIPTION
**Pull request by calebpanza**
## Motivation
This is a composite component that builds on the Avatar and typography components and is a key piece of our social experiences.

## Testing
Stories are written and test coverage is at 100%

| Basic | Micro |
| --- | --- |
| Simulator Screen Shot - iPad Air (5th generation) - 2022-09-29 at 09 56 31 | Simulator Screen Shot - iPad Air (5th generation) - 2022-09-29 at 09 56 24 |

### CODE REVIEW

**Comment by calebpanza** [comment]
I meant to mark this as a Draft. Please don't merge yet

File: templates/mobile/src/ui-kit/components/Byline/Byline.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/**
+ * Byline.js
+ *
+ * @description
```
Comment (conrad-vanl) [inline templates/mobile/src/ui-kit/components/Byline/Byline.js:4]: There's no description 😭

**Review by conrad-vanl — APPROVED** [review]

File: templates/mobile/src/ui-kit/components/Byline/Byline.js (hunk 1 above)
Comment (calebpanza) [reply templates/mobile/src/ui-kit/components/Byline/Byline.js]: Added! hahah

File: templates/mobile/src/ui-kit/components/Byline/Byline.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /**
  * Byline.js
  *
- * @description
+ * A component that renderes an Avatar and user's name given their profile in a standard of micro style. Optionally pass in a label. This is a really great foundation for building social experiences
```
[the remark this answers is not in the document]
Comment (conrad-vanl) [reply templates/mobile/src/ui-kit/components/Byline/Byline.js:4]: Screen Shot 2022-09-29 at 9 47 04 AM

[the remark this answers is not in the document]
Comment (calebpanza) [reply templates/mobile/src/ui-kit/components/Byline/Byline.js:4]: I like that idea. I'll put out another PR real quick that adds links for all existing components

[the remark this answers is not in the document]
Comment (conrad-vanl) [reply templates/mobile/src/ui-kit/components/Byline/Byline.js:4]: https://github.com/ApollosProject/apollos-apps/blob/master/packages/apollos-ui-kit/src/ContentTitles/ContentTitles.js#L14-L28 don't need to do this exactly but when I started porting components from figma I'd first start with the link and a list to all the props / features represented in figma and check off as I added them

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**